\documentclass[12pt]{article}
\usepackage{geometry}                % See geometry.pdf to learn the layout options. There are lots.
\geometry{a4paper,
 total={170mm,257mm},
 left=20mm,
 top=20mm,
 bottom=40mm}                   % ... or a4paper or a5paper or ... 
%\geometry{landscape}                % Activate for for rotated page geometry
\usepackage[parfill]{parskip}    % Activate to begin paragraphs with an empty line rather than an indent
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{epstopdf}
\usepackage{float}
\usepackage{hyperref}
\hypersetup{%
  colorlinks=true,% hyperlinks will be coloured
  linkcolor=blue,% hyperlink text will be green
}
\DeclareGraphicsRule{.tif}{png}{.png}{`convert #1 `dirname #1`/`basename #1 .tif`.png}

\usepackage[backend=bibtex, sorting=none, style=chicago-authordate]{biblatex}
\setlength\bibitemsep{\baselineskip}
\addbibresource{exampleRef.bib}

%% LOGOS
\usepackage{fancyhdr}
\pagestyle{fancyplain} % use fancy for all pages except chapter start
\lhead{\includegraphics[height=1.3cm, width=2.2cm]{Logos/NBIS-logo-orange-text.png}} % left logo
\rhead{\includegraphics[height=1.3cm, width=4cm]{Logos/SciLifeLab-logo.jpg}} % right logo
\renewcommand{\headrulewidth}{0pt} % remove rule below header

%% DEFINE VARIABLES
\newcommand{\staff}{Olga Dethlefsen}
\newcommand{\staffEmail}{olga.dethlefsen@nbis.se}
\newcommand{\staffWeb}{http://nbis.se/about/staff/olga-dethlefsen/}

\newcommand{\coordinator}{}

\newcommand{\affilations}{National Bioinformatics Infrastructure Sweden, Science for Life Laboratory, Stockholm University}
\newcommand{\supportWeb}{http://nbis.se/support/support.html}
\newcommand{\uppmaxWeb}{http://www.uppmax.uu.se/support/faq/general-miscellaneous-faq/acknowledging-uppmax--snic--and-uppnex/}

\newcommand{\noIssue}{\#9999}
\newcommand{\noUppmax}{b29999}


%% BEGIN DOCUMENT
%\date{}
\usepackage{Sweave}
\begin{document}
\pagestyle{fancy}
\input{9999_report-concordance}

%% TITLE PAGE
\title{Example}
\author{}
\maketitle
\thispagestyle{fancy}

\vspace{2cm}
\begin{center}
\begin{tabular}{l r}
Issue number: & {\noIssue} \\
Request by: & Jan User <jan.user@ki.se> \\ 
Principal Investigator: & Maria Investigator <maria.investigator@ki.se> \\
Organisation: & Karolinska Insitutet \\
NBIS staff: & {\staff}<{\staffEmail}>
\end{tabular}
\end{center}

%% TABLE OF CONTENTS
\newpage
\tableofcontents

%% SUPPORT REQUEST
\newpage
\section{Support request}

Include support request as sent by the group and seen in Redmine (for reference purposes)

\section{Important practical information}
\subsection{Data responsibility}
\label{DataRes}
\footnotesize
\begin{description}
\item [NBIS \& Uppnex] Unfortunately, we do not have resources to keep any files associated with the support request. We kindly suggest that you store safely the results delivered by us. In addition, we kindly ask that you remove the files from UPPMAX/UPPNEX. The main storage at UPPNEX is optimized for high-speed and parallel access, which makes it expensive and not the right place for longer time archiving. Please consider others by not taking up the expensive space.
  \item[Long-term backup] The responsibility for data archiving lies with universities and we recommend asking your local IT for support with long-term data archiving. Also a newly established \href{https://www.scilifelab.se/data/}{Data Office} at SciLifeLab may be of help to discuss other options. 
  %\item[Data submission] NBIS can help with raw sequencing data submission to ENA. Please contact our data manager, Niclas Jareborg, niclas.jareborg@nbis.se for more details. 
  \end{description}

\subsection{Acknowledgments}
\label{acknowledgments}
If you are presenting the results in a paper, at a workshop or conference, we kindly ask you to acknowledge us.
\begin{description}
\item[NBIS staff] are encouraged to be co-authors when this is merited in accordance to the ethical recommendations for authorship, e.g. {\href{http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html}{ICMJE recommendations}}. If applicable, please include {\href{\staffWeb}{{\staff}, \affilations}} as co-author. In other cases, NBIS would be grateful if support by us is acknowledged in publications according to this example: \href{https://bils.se/resources/support.html}{"Support by NBIS (National Bioinformatics Infrastructure Sweden) is gratefully acknowledged."}
\item[Uppmax] kindly asks you to acknowledge UPPMAX and SNIC. If applicable, please add: {\href{\uppmaxWeb}{The computations were performed on resources provided by SNIC through Uppsala Multidisciplinary Center for Advanced Computational Science (UPPMAX) under Project SNIC {\noUppmax}.}}
% \item[Mosler] if applicable, according to the resource user agreement for the Mosler system for sensitive data, please acknowledge Mosler by adding \href{https://nbis.se/infrastructure/mosler.html}{Support by NBIS (National Bioinformatics Infrastructure Sweden) for the use of the Mosler system for sensitive data is gratefully acknowledged}


\item[NGI Stockholm] In any and all publications based on data from NGI Sweden, the authors must acknowledge SciLifeLab, NGI and Uppmax, like so: \href{https://ngisweden.scilifelab.se/info/faq#how-do-i-acknowledge-ngi-in-my-publication}{The authors would like to acknowledge support from Science for Life Laboratory, the National Genomics Infrastructure, NGI, and Uppmax for providing assistance in massive parallel sequencing and computational infrastructure.}
\end{description}

%% WORK LOG SECTION
\section{Work log}
A brief project history containing key points
\begin{description}
\item[2015-09-15] first meeting with Jan to discuss experimental design, available data and desired results.	As first results, Jan would like to receive lists of differentially expressed (DE) genes between the two time points for the two groups
\item[2015-10-09] meeting with Jan to go over the DE results. Agreed that Jan will go over the DE results and try running gene set enrichment analyses using DAVID website.
\item[2015-11-06] run and emailed Jan the exon usage results for the 4 comparisons
\item[2015-12-01] run and emailed Jan the motif discovery Homer results
\item[2015-12-10] meeting with Jan to discuss the additional 
\item[2015-12-20] final results and report delivered
\end{description}


%% MATERIALS AND METHODS SECTION
\section{Materials and Methods}
Include Materials and Methods as in any scientific work

\section{Results}
Include results

\section{Where to go next}
Optionally include things to help the group to use the results, e.g. recommendation of online tools that could be used to inspect the data, or tutotrials to understand in details data analysis etc.

\section{Support project closing procedures}
You should soon be contacted by one of our managers, Jessica Lindvall <jessica.lindvall@nbis.se> or Henrik Lantz <henrik.lantz@nbis.se>, with a request to close down the project in our internal system and for invoicing matters. If we do not hear from you within \textbf{30 days} the project will be automatically closed and invoice sent. Again, we would like to remind you about data responsibility and acknowledgements, see \nameref{DataRes} and \nameref{acknowledgments}. 

You are naturally more than welcome to come back to us with further data analysis request at any time via \href{http://nbis.se/support/support.html}{http://nbis.se/support/support.html}. Thank you for using NBIS and all the best for future research. 

\end{document}  
